src/resources/i18n/translations/en.json

```diff
@@ -243,7 +244,7 @@
   "Custom web domain": "Web domain",
   "Custom web domain subtext": "Set custom domains to use an alternate server for your app data.",
   "Dark Mode": "Dark",
-  "Default downloaded episode limit for each podcast": "Episode download-limit for each podcast",
+  "Default downloaded episode limit for each podcast": "Episodes to keep for each podcast",
```

Comment:
I'm thinking maybe "Maximum downloaded episodes for each podcast"?

Reply:
"Max downloaded episodes of each podcast" then, but I think it is more complex(?)